highfive/newpr.py

```diff
@@ -36,7 +36,7 @@
 review_with_reviewer = 'r? @%s\n\n(rust_highfive has picked a reviewer for you, use r? to override)'
 review_without_reviewer = '@%s: no appropriate reviewer found, use r? to override'
 
-reviewer_re = re.compile("\\b[rR]\?[:\- ]*@([a-zA-Z0-9\-]+)")
+reviewer_re = re.compile("\\b[rR]\?[:\- ]*@(?:([a-zA-Z0-9\-]+)/)?([a-zA-Z0-9\-]+)")
```

Comment:
We only want to match rust-lang, don't we?
```suggestion
reviewer_re = re.compile("\\b[rR]\?[:\- ]*@(rust-lang/)?([a-zA-Z0-9\-]+)")
```

Reply:
I think we have also configs for rust-embedded, rust-nursery and rust-deprecated.